_posts/2018-05-24-security-session-highlights-from-hkg18.md

```diff
@@ -0,0 +1,29 @@
+---
+author: op-tee
+title: Security Session Highlights from HKG18
+date: 2018-05-24 12:00:00
+categories:
+- Blog
+image:
+    featured: true
+    path: /assets/images/hkg18-banner.jpg
```

Comment:
Can this be the same path as where the image was added to the repo?

Reply:
yep